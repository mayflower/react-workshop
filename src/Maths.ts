export default class Maths {
  sum(a: number, b: number) {
    return a + b;
  }
}