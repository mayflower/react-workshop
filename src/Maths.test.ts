import Maths from './Maths'

it('Adds up two more numbers', () => {
  let m = new Maths;
  [[1, 2], [22, 4], [5, 6], [7, 8], [9, 100]].forEach((num) => {
    expect(m.sum(num[0], num[1])).toMatchSnapshot();
  })
});
