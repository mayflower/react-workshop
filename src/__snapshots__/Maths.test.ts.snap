// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Adds up two more numbers 1`] = `3`;

exports[`Adds up two more numbers 2`] = `26`;

exports[`Adds up two more numbers 3`] = `11`;

exports[`Adds up two more numbers 4`] = `15`;

exports[`Adds up two more numbers 5`] = `109`;
